node/pkg/node/node_test.go

```diff
@@ -841,7 +841,7 @@ func runConsensusTests(t *testing.T, testCases []testCase, numGuardians int) {
 	<-rootCtx.Done()
 	assert.NotEqual(t, rootCtx.Err(), context.DeadlineExceeded)
 	zapLogger.Info("Test root context cancelled, waiting 10ms for everything to shut down properly...")
```

Comment:
Log message doesn't match the actual sleep time.

Reply:
Will fix!